src/Apps/Home/Components/HomeAuctionLotsForYouRail.tsx

```diff
@@ -0,0 +1,140 @@
+import { Shelf, Skeleton } from "@artsy/palette"
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "System/useSystemContext"
+import { useTracking } from "react-tracking"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import {
+  ShelfArtworkFragmentContainer,
+  ShelfArtworkPlaceholder,
+} from "Components/Artwork/ShelfArtwork"
+import {
+  ActionType,
+  ClickedArtworkGroup,
+  ContextModule,
+  OwnerType,
+} from "@artsy/cohesion"
+
+import { HomeAuctionLotsForYouRail_artworksForUser$data } from "__generated__/HomeAuctionLotsForYouRail_artworksForUser.graphql"
+import { HomeAuctionLotsForYouRailQuery } from "__generated__/HomeAuctionLotsForYouRailQuery.graphql"
+import { extractNodes } from "Utils/extractNodes"
+
+interface HomeAuctionLotsForYouRailProps {
+  artworksForUser: HomeAuctionLotsForYouRail_artworksForUser$data
+}
+
+const HomeAuctionLotsForYouRail: React.FC<HomeAuctionLotsForYouRailProps> = ({
+  artworksForUser,
+}) => {
+  const { trackEvent } = useTracking()
+
+  const artworks = extractNodes(artworksForUser)
+
+  if (!artworks || artworks?.length === 0) {
+    return null
+  }
+
+  return (
+    <Shelf>
+      {artworks.map((artwork, index) => {
+        if (!artwork) {
+          return <></>
+        }
+
+        return (
+          <ShelfArtworkFragmentContainer
+            artwork={artwork}
+            key={index}
+            contextModule={ContextModule.auctionLots}
+            lazyLoad
+            onClick={() => {
+              //TODO: add tracking for Auction Lots For You
+              const trackingEvent: ClickedArtworkGroup = {
+                action: ActionType.clickedArtworkGroup,
+                context_module: ContextModule.auctionLots,
```

Comment:
**suggestion / non-blocking**: Probably best to add a new context module within the context of this PR. It's easy to forget about otherwise.

Maybe two new Cohesion values which can replace this more generic one?
- `auctionLotsForYouRail`
- `topAuctionLotsRail`

cc/ @abhitip fyi and if you have suggestions on the naming ☝🏼

Reply:
Hey, yes that's the plan - that's why I'm keeping this PR as a Draft still until we clarify all the missing info.

Reply:
Yes those values sound great!

Reply:
The new values [in Cohesion](https://github.com/artsy/cohesion/pull/487/files) are `lotsForYouRail` for recommended auction lots and `topAuctionLotsRail` for curators picks.